src/elements/content-uploader/__tests__/UploadState.test.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { render, screen } from '../../../test-utils/testing-library';
+
+import UploadState, { UploadStateProps } from '../UploadState';
+
+import { VIEW_ERROR, VIEW_UPLOAD_EMPTY, VIEW_UPLOAD_IN_PROGRESS, VIEW_UPLOAD_SUCCESS } from '../../../constants';
+
+describe('elements/content-uploader/UploadState', () => {
+    const getWrapper = (props: Partial<UploadStateProps>) =>
```

Comment:
nit:
for RTL, we've been calling this `renderComponent` since it's not returning the wrapper anymore